pal/utils/src/index.ts

```diff
@@ -271,6 +271,18 @@ export function clearTimeoutRAF (id: number): void {
     }
 }
 
+/**
+ * Invoke a function in next frame.
+ * @param callback The function to be invoked next frame.
+ * @param p1 The first parameter passed to `callback`.
+ * @param p2 The seconde parameter passed to `callback`.
+ */
+export function callInNextTick (callback: AnyFunction, p1?: unknown, p2?: unknown): void {
```

Comment:
Why it is part of pal?

Reply:
because the implementation of `callInNextTick` rely on `RAF`(requestAnimationFrame) , which should be part of PAL

Reply:
The drawing process is triggered by RAF. So every codes that needs to be invoked every frame rely on RAF, for example, scheduleOnce. So i think think it is not a reasonable reason to implement `callInNextTick ` in pal.